config.libsonnet

```diff
@@ -7,6 +7,12 @@
     clusterUtilizationAlertTime: '5m',
     clusterStateAlertTime: '10m',
     monQuorumAlertTime: '15m',
+    osdDownAlertTime: '5m',
+    osdDataRecoveryAlertTime: '2h',
+    osdDiskAlertTime: '1m',
+    osdDataRebalanceAlertTime: '15s',
+    osdDataRecoveryInProgressAlertTime: '30s',
```

Comment:
Same alphabetical order thing here as well

Reply:
Ack. Will do that.